nfc/CHANGELOG.rst

```diff
@@ -30,6 +30,8 @@ Bug fixes
 
  * Fixed handling R(NAK) frame after sending R(ACK) frame.
    The Tag should respond with last transmitted R(ACK) frame.
+ * Fixed race condition when the WTX frame is scheduled and library tries to send data chunk.
+ * Fixed possible memory overwrite in the NFC T4T library when the Reader device sends an incorrect APDU Update command.
```

Comment:
```suggestion
 * Fixed a possible memory overwrite in the :ref:`lib_nfc_t4` library when the reader device sends an incorrect APDU Update command.
```

Reply:
Or should the link rather go to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/nfc/doc/type_4_tag.html? If yes, then :ref:`type_4_tag`.